### ISSUE TITLE
Poor kerning with subscripts, capital V, and an overmark

### ISSUE THREAD

**Issue by jason-s** [mathjax/MathJax#2419]
`\tilde{V}_d` looks bad; too much spacing before the d, and it's not consistent with TeX. Other letters like `\tilde{R}_d` look ok.

I'm on Windows 10 but this is observable with Chrome, Firefox, and Opera.

Live example:

https://codepen.io/jason-s/full/LYpLRXO

Source code:

```
<html>
<head>
<script>
MathJax = {
  tex: {
    inlineMath: [['$', '$'], ['\\(', '\\)']]
  },
  svg: {
    fontCache: 'global'
  }
};
</script>
<script type="text/javascript" id="MathJax-script" async
  src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js">
</script>
</head>
<body>
  <p>
Here is the issue: $V_d \quad \tilde{V}_d \quad \hat{V}_d \quad \hat{R}_d$ 
  </p>
  
<p>
Putting the subscript inside the overmark looks better for kerning, but is not a good idea: $\tilde{V_{sandwich}} \quad \tilde{V}_{sandwich}$
  </p>  
</body>
</html>
```

----

TeX screenshot:

![image](https://user-images.githubusercontent.com/738893/80431341-11096b00-88a6-11ea-9218-2f3e96e36023.png)

```
\documentclass[border=1mm]{standalone}
\usepackage{tikz}
\begin{document}
\begin{tikzpicture}
\node[scale=5] at (0,0) {$\tilde{V}_d \quad \tilde{R}_d$};
\end{tikzpicture}
\end{document}
```

Screen capture of Chrome with my Codepen example at 300% size:

![image](https://user-images.githubusercontent.com/738893/80431404-339b8400-88a6-11ea-83c8-8e8c59718b58.png)

**Comment by dpvc** [comment]
TeX does treat some accents in a special way that is not reflected in MathJax, and is leading to the results that you see here.  I will mark this as a feature request so that it can be considered for a future update.

**Comment by dpvc** [comment]
This has been fixed in 3.2.
